### PR TITLE
About implementation

### PR DESCRIPTION
**Pull request by Ne-anna**
In this PR I have added all needed images to the project. Created project part - about.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
 function App() {
+  const getData = async () => {
+    const requestURL =
+      "https://raw.githubusercontent.com/Ne-anna/renome-typescript/about-implementation/src/db.json";
+    const request = await fetch(requestURL);
+
+    return request.json();
+  };
+
+  const [data, setItems] = useState<any>();
```
Comment (ReinisZogots) [inline src/App.tsx]: Try not to use type Any.

File: src/App.tsx [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
+
   return (
     <div>
       <header>
-        <p>i need coffee</p>
+        <p>gribu kafiju un est</p>
       </header>
+      <main>
+        <About
+          title={data?.about?.title}
```
Comment (ReinisZogots) [inline src/App.tsx]: Try to do without "?"

**Review by Jalal999 — APPROVED** [review]

**Review by Jose-Hash-Dev — APPROVED** [review]

**Review by ReinisZogots — APPROVED** [review]

File: src/App.tsx [hunk 3]
```diff
@@ -1,11 +1,38 @@
-import React from "react";
+import About from "./Components/Organisms/About/about";
+import { useEffect, useState } from "react";
+import { defaultData } from "./data";
 
 function App() {
+  const getData = async () => {
```
Comment (ghost) [inline src/App.tsx:6]: have explicit return type if possible

File: src/App.tsx [hunk 4]
```diff
@@ -1,11 +1,38 @@
...
+    const requestURL =
+      "https://raw.githubusercontent.com/Ne-anna/renome-typescript/about-implementation/src/db.json";
+    const request = await fetch(requestURL);
+
+    return request.json();
+  };
+
+  const [data, setItems] = useState(defaultData);
```
Comment (ghost) [inline src/App.tsx:14]: avoid using < any > (inferred currently)
add type like: const [data, setItems] = useState< YourType >(defaultData);
also naming to [data, setData] or [items, SetItems] would be bit more suitable

edit: markdown

File: src/data.ts [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+  menu: Menu[];
+  backButtonTitle: BackButtonTitle;
+  carousel: Carousel[];
+  carouselButtons: CarouselButtons;
+  about: About;
+  gallery: Gallery;
+  footer: Footer;
+}
+
+export const defaultData = {
```
Comment (ghost) [inline src/data.ts:100]: please add type, avoid any

**Pull request merged**